src/logic/calculate.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
+class Components extends Component {
+  render() {
+    return (
+      <div className="button">
+        <button name="C" onClick={(e) => this.props.onClick(e.target.name)}>AC</button>
+        <button name="+/-" onClick={(e) => this.props.onClick(e.target.name)}>+/-</button>
```

Comment:
- [ ] You should Import calculate.js in your calculator component so it can work